src/sum_from_string.py

```diff
@@ -0,0 +1,16 @@
+"""A function to sum numbers found in a string."""
+
+
+def sum_from_string(string):
+    """Sum numbers found in a string."""
+    new_l = []
```

Comment:
Both parts of your solution can be simplified into nice-looking list comprehensions. Try them out, they're great!